src/mmw/js/src/data_catalog/templates/cuahsiSearchResult.html

```diff
@@ -1,22 +1,19 @@
 <h3 class="resource-title">
-    {{ id }}
+    <a href="{{service_url}}" target="_blank">{{ service_code }}</a> {{ id | split(":", 1) }}
 </h3>
 <div class="resource-detail">
     <i class="fa fa-map-marker"></i> {{ title }}
```

Comment:
Here's what this looks like now:

![image](https://user-images.githubusercontent.com/1042475/29149924-3a4155d8-7d45-11e7-9b2f-2df7810e37e7.png)

Which went to:

![image](https://user-images.githubusercontent.com/1042475/29149935-4fb56102-7d45-11e7-8f9e-10f4623d32c5.png)